config_example.yml

```diff
@@ -0,0 +1,22 @@
+asset_store:
+  synapse:
+    config_basename: ".synapseConfig"
+    manifest_basename: 'synapse_storage_manifest'
+    master_fileview_id: 'syn23643253'
+    manifest_folder: 'manifests'
+
+manifest:
+  title: 'example'
+  data_type:
+    - 'Biospecimen'
+    - 'Patient'
+
+model:
+  location: 'tests/data/example.model.jsonld'
+  file_type: 'local'
+
+google_sheets:
+  service_acct_creds_synapse_id: 'syn25171627'
+  service_acct_creds_basename: "schematic_service_account_creds.json"
+  master_template_id: null
```

Comment:
why master_template_id is null?

Reply:
So I [asked](https://sagebionetworks.slack.com/archives/C050YD75QRL/p1683831221912009) about this in slack and idn't get any reponses. I asked @milen-sage about in our one-on-one and this is what he wanted to do.

Reply:
I'm going to change this back to the default being the old template id. It looks like making the default be null needs to be a separate issue.

Reply:
I think currently, the master_template_id is hard-coded, and i personally think changing it to null would be confusing since people might think we are not using the master_template_id

Reply:
Before I changed things back that's exactly what was happening.